include/hipSYCL/sycl/libkernel/math.hpp

```diff
@@ -117,6 +117,9 @@ namespace sycl {
   HIPSYCL_DEFINE_FLOATN_TRINARY_MATH_FUNCTION(func, func)
 #endif
 
+HIPSYCL_DEFINE_FLOATING_POINT_OVERLOAD(abs, fabsf, fabs)
+HIPSYCL_DEFINE_FLOATN_MATH_FUNCTION(abs, abs)
```

Comment:
We already have fabs for floating point types. The spec does not define abs for floats as far as I can see.